packages/core/src/button/Button.css

```diff
@@ -161,20 +161,21 @@
   outline-offset: var(--button-focused-outlineOffset);
 }
 
-/* Pseudo-class applied to the root element on focus when Button is not active */
-.uitkButton:focus-visible:not(:active):not(.uitkButton-active) {
+/* Pseudo-class applied to the root element on focus when Button is not active and not disabled
+   (focus ring will still be visible if button is focusable when disabled) */
+.uitkButton:focus-visible:not(:active):not(.uitkButton-active):not(.uitkButton-disabled) {
   background: var(--uitkButton-background-hover, var(--button-background-hover));
   color: var(--uitkButton-text-color-hover, var(--button-text-color-hover));
 }
 
 /* Pseudo-class applied to the root element on hover when Button is not active or disabled */
-.uitkButton:hover {
+.uitkButton:hover:not(.uitkButton-disabled) {
```

Comment:
The usage of :not means the css is more difficult to override but for now this seems fine.